src/poetry/puzzle/provider.py

```diff
@@ -664,7 +664,9 @@ def complete_package(
                 dependencies.append(deps[0])
                 continue
 
-            self.debug(f"<debug>Duplicate dependencies for {dep_name}</debug>")
+            self.debug(
+                f"<debug>Duplicate dependencies for {dep_name}</debug>"  # noqa: G004
```

Comment:
Does it make sense to use `flake8-logging-format` if we suppress most of the errors?

I'm also a bit ambivalent about disallowing f-strings in logging messages. We neither have shown that this is a performance issue nor we have a logging handler that like to process the arguments.

Reply:
Oh, I didn't recognize it is `self.debug` and not `logger.debug` :see_no_evil: